content/en/docs/reference/glossary/resource-driver.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Resource Driver
+id: resource-driver
+date: 2023-10-16
+full_link: 
+short_description: >
+  Software extensions to let Pods access devices that need vendor-specific initialization or setup
+aka:
+tags:
+- extension
+- fundamental
+---
+ Resource Driver is responsible for allocation of non-native resources requested by
+{{< glossary_tooltip term_id="resource-claim" text="Resource Claims">}}.
+
+<!--more-->
+
+Typically consists of one controller {{< glossary_tooltip term_id="pod" text="Pod ">}} and many
+kubelet-plugin Pods. Controller allocates hardware resources requested by Resource Claim.
+Kubelet-plugin discovers supported hardware devices, advertises them to controller, and prepares,
+unprepares the allocated resources when {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}
+is preparing to start or has stopped the Pod.
+
+There can be multiple {{< glossary_tooltip term_id="resource-class" text="Resource Classes">}}
+associated with one Resource Driver, typically in such case they have different
+{{< glossary_tooltip term_id="resource-class-parameters" text="Resource Class Parameters">}}
+that customize resources allocation process.
+
+
+See
+[Dynamic Resource Allocation](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/3063-dynamic-resource-allocation)
+for more information.
```

Comment:
This should link to the Kubernetes docs on DRA (which might then mention the KEP as further reading).

Reply:
Done.